README.md

```diff
@@ -57,8 +59,8 @@ forecasting - which is notorious for the size and complexity of its workflows.
 Cylc-8.0.0 will be released in 2021. Until then we recommend the latest
 Cylc 7 versions for production use.
 
-[Installation](https://cylc.github.io/cylc-doc/8.0a2/html/installation.html) |
-[Documentation - in development](https://cylc.github.io/cylc-doc/8.0a2/)
+[Installation](https://cylc.github.io/cylc-doc/current/html/installation.html) |
+[Documentation - in development](https://cylc.github.io/cylc-doc/current/html/index.html)
```

Comment:
Shouldn't be a link to the Cylc 7 docs?

Reply:
(maybe to the cylc-doc repository instead, for the moment)

Reply:
Was the link not correct before? 8.0a2 is the most recent docs release.

Reply:
(we could adopt the tags `stable` and `latest` for docs releases if desired, at present we only have `current` which means "stable", these docs haven't been released yet so can be changed).

Reply:
> Was the link not correct before? 8.0a2 is the most recent docs release.

Ah yes it was correct - I forgot we had deployed 8.0a2 docs to the GH Pages site.

Reply:
I've put up a PR for latest + stable: https://github.com/cylc/cylc-doc/pull/199

Reply:
Links still point the Cylc7 docs.